src/art/__init__.py

```diff
@@ -11,19 +12,19 @@
 
 from . import dev
 from .api import API
+from .skypilot import SkyPilotAPI
 from .gather import gather_trajectories, gather_trajectory_groups
 from .model import Model, TrainableModel
 from .trajectories import Trajectory, TrajectoryGroup
 from .types import Messages, MessagesAndChoices, Tools, TrainConfig
-from .local import LocalAPI
 from .utils import retry
 
 __all__ = [
     "dev",
     "gather_trajectories",
     "gather_trajectory_groups",
     "API",
-    "LocalAPI",
+    "SkyPilotAPI",
```

Comment:
I'm not sure we want to include this here/encourage people to import directly from the top-level `art` object. Skypilot is a heavy dependency that a lot of folks won't need; I'd rather not import it into the top-level object so in the future we can ship the core without it, and if someone does want to use our skypilot support they can run `uv add openpipe-art[skypilot]`.